03/app.test.js

```diff
@@ -1 +1,28 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('return 0 when min is not set', () => {
+    expect(randomNumber(-1)).toBe(0);
+    expect(randomNumber(undefined)).toBe(0);

```

Comment:
Raczej rozdzieliłbym ten testy. 
Opis mówi, że parametr nie jest ustawiony, a `-1` to konkretna wartość